### PR TITLE
scratchblocks-v3 using WIP and PLSEXPLAIN

### LINKED ISSUE #438: Forum Scratchblocks Injection
This would be an extension that injected the new scratchblocks into the forums.

### PR DESCRIPTION
**Pull request by apple502j**
Resolves #438

### CODE REVIEW

**Comment by RedGuy12** [comment]
Can your rename the translation file to clarify it's the translations for scratchblocks?

File: libraries/translations-all-v3.5.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
```
Comment (WorldLanguages) [inline libraries/translations-all-v3.5.js:1]: Should this be added to the list of libraries?

Comment (apple502j) [reply libraries/translations-all-v3.5.js:1]: Prob not

Comment (WorldLanguages) [reply libraries/translations-all-v3.5.js:1]: Why not?

Comment (apple502j) [reply libraries/translations-all-v3.5.js:1]: It's part of scratchblocks, just 2 files

**Comment by WorldLanguages** [comment]
I can't get asap.js to run consistently before vanilla 2.0 scratchblocks is ran, we need to find a better way to do this

**Comment by GrahamSH-LLK** [comment]
> I can't get asap.js to run consistently before vanilla 2.0 scratchblocks is ran, we need to find a better way to do this

Same, I could consistently reproduce it not working at all.

**Pull request closed without merging**